src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,86 @@
+package duke;
+
+import java.time.LocalDate;
+
+/**
+ * Deals with making sense of the user command
+ */
+public class Parser {
+    /**
+     * Deals with parsing the command and do some reaction
+     * @param input user command input
+     * @param taskList a list of tasks
+     * @return if the command required to exit
+     * @throws DukeException exceptions that are possible to occur in Duke
+     */
```

Comment:
For Javadoc, there should be an empty line between description and parameter section